packages/clay-management-toolbar/src/ActiveState.tsx

```diff
@@ -0,0 +1,252 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+import Item from './Item';
+import ItemList from './ItemList';
+import React from 'react';
+import {ClayCheckbox} from '@clayui/form';
+import {ClayDropDownWithBasicItems} from '@clayui/drop-down';
+import {IItem} from '@clayui/drop-down';
+import {IPropsBase} from './types';
+import {sub} from '@clayui/shared';
+
+interface IActionItems extends IItem {
+	quickAction?: boolean;
+}
+
+interface IProps extends IPropsBase {
+	actionItems?: Array<IActionItems>;
+	clearSelectionURL?: string;
+	onClearSelectionClick?: (
+		e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+	) => void;
+	onQuickActionClick?: (
+		e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+	) => void;
+	onSelectAllClick?: (
```

Comment:
Rather than re-declare this type over and over, we could cache it to a type. `type ClickCallback = //...`

Reply:
Oh yeah